cps/CP21.md

```diff
@@ -0,0 +1,50 @@
+---
+cp: 16
+title: 2022-12-16 Updating Tinlake Rewards Allocation
+authors: Governance and Coordination Group (ImdioR, Orhan)
+contributors: Governance and Coordination Group (ImdioR, Orhan)

```

Comment:
do we need "contributors" when they are the same as "authors"?

Reply:
hm...i think we can just follow the template and in case we would like to remove it we can do this in CP1.2?

Reply:
I removed it in the RWA Market - it only makes sense to have _contributors_, if they are different than the authors.